docs/tutorials/gs_continuous_space.ipynb

```diff
@@ -0,0 +1,1210 @@
+{
```

Comment:
<p>maybe let's just set m=1 also in the latex? or we write explicit units for the X^2 part </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>